### PR TITLE
SignaturePad - fromUrl - Resizing image

### LINKED ISSUE #5411: [Bug]: SignaturePad - fromUrl - Resizing image
### Blazorise Version

1.5.0

### What Blazorise provider are you running on?

Bootstrap5

### Link to minimal reproduction or a simple code snippet

https://bootstrap5demo.blazorise.com/tests/signaturepad 

watch the canvas and the preview

### Steps to reproduce

1) browse to https://bootstrap5demo.blazorise.com/tests/signaturepad 
2) draw on the canvas 

### What is expected?

the canvas image is the same as what was drawn on the canvas.

### What is actually happening?

the canvas image is resized larger.

### What browsers do you see the problem on?

Chrome

### Any additional comments?

_No response_

### PR DESCRIPTION
**Pull request by sks-ttaylor**
## Description

Closes #5411

## How Has This Been Tested?
Locally using Blazorise.Demo.Bootstrap5

## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Checklist:

- [ x] The PR is submitted to the correct branch (`master` for dev, `rel-1.x` for maintenance).
- [ ] My code follows the code style of this project.
- [ ] I've added relevant tests.

### CODE REVIEW

File: Source/Extensions/Blazorise.SignaturePad/wwwroot/signaturepad.js [hunk 1]
```diff
@@ -3,6 +3,10 @@ import "./vendors/sigpad.js?v=1.5.0.0";
 
 const _instances = [];
 
+function GetRatio() {
```
Comment (David-Moreira) [inline Source/Extensions/Blazorise.SignaturePad/wwwroot/signaturepad.js:6]: In javascript we use the convention of camelcase for functions. `getRatio()`

**Comment by sks-ttaylor** [comment]
I will update my branch...

________________________________

**Comment by stsrki** [comment]
Can I please ask you to reopen new PR from a differently named branch? We don't allow same branchnames targeting our repos as there were problems in the past.

image

Just go with something like `rel-1.5-signaturepad-fix`.

**Comment by sks-ttaylor** [comment]
will do..

________________________________

**Comment by sks-ttaylor** [comment]
new sks-ttaylor branch "rel-1.5-signaturepad-fix" created
please delete the previous PR "SignaturePad - fromUrl - Resizing image #5413".
new PR created "Rel 1.5 signaturepad fix #5414".

________________________________

**Pull request closed without merging**